charts/opensearch-dashboards/values.yaml

```diff
@@ -59,13 +59,13 @@ securityContext:
   runAsUser: 1000
 
 config: {}
-  ## Default OpenSearch Dashboards configuration from docker image of Dashboards
+  # Default OpenSearch Dashboards configuration from docker image of Dashboards
```

Comment:
Two ## better indicate a command. Is this a linting thing?

Reply:
Yes, it's a linting thing.

Reply:
Space wasn't the concern here, you removed ## -> #. I'm wondering if that part is intentional?

Reply:
The `##` needed to be `# ` to avoid the yaml lint error.